### ISSUE TITLE
Minimal os requirements for macOS

### ISSUE THREAD

**Issue by circleapps** [Kagami/mpv.js#47]
Hello, one of my user found mpv.js displaying "Couldn't load plugin" on OSX 10.11.6.

So what is the minimal os requirements of mpv.js? Thanks.

**Comment by Kagami** [comment]
Should be 10.9+, see https://github.com/Kagami/boram/issues/11
The error might be caused by something else.

**Comment by circleapps** [comment]
Kagami,

I am trying to reproduce this issue with the example player. 

When I started to install mpv with `brew install mpv` to the a machine on OSX 10.11.6,  I got this error:

```
CMake Error: No source or binary directory provided

Do not report this issue to Homebrew/brew or Homebrew/core!

These open issues may also help:
Update graphite2 to reference src dir in cmake https://github.com/Homebrew/homebrew-core/pull/36376

Error: You are using macOS 10.11.
We (and Apple) do not provide support for this old version.
```

Looks like brew script for mpv has dropped support for this version of OSX.

Do you have any other way to install mpv to support the running of mpv.js? (I tried the pre-build version of mpv, it did not work).

Thanks.

**Comment by Kagami** [comment]
For boram I built the mpv and its dependencies on Sierra (10.12) and copy dependent libs using [this script](https://github.com/Kagami/boram/blob/master/scripts/make-mac64-deps.sh).

**Comment by circleapps** [comment]
Kagami, I found your script used brew to build mpv too, so I believe it will get the same error if I do it on 10.11.6. Because the HEAD version of mpv.rb has dropped 10.11 support:

```
  bottle do
    rebuild 1
    sha256 "a7e67ef6a0e0b71320164167e14c9a1ce0d237a9ff822012372c2315ea08b159" => :mojave
    sha256 "11cc6bad21c68054d6ea85fc0b91ba5e4f945107b93855c3a54933ceb81651ff" => :high_sierra
    sha256 "30eacc0d71f4f3ee3be680aadc0a1426f3f8f8dd166e3198e3b475ae26bb0f22" => :sierra
  end
```

Then I  switched to the older version of mpv.rb to build mpv (it supports 10.11),  but unfortunately the dependencies of mpv have the same issue (HEAD version does not support 10.11), the build process failed again.

So my final solution would be creating my own mpv brew build script to support 10.11. I will let you know the result once I fixed it.

Thanks.

**Comment by Kagami** [comment]
Correct, I build deps on Sierra, then use the script to collect .dylib deps and ship everything together to target platform and it works. At least it perfectly worked for 10.9 few years ago.

**Comment by circleapps** [comment]
Found my way to create a custom build for macOS 10.11.

1.  Create your own brew formulas for both ffmpeg and mpv like  this one:
https://github.com/varenc/homebrew-ffmpeg

The HEAD version of the official mpv and ffmpeg brew formulas don't have options for you to customize, that is why I create my own ones.

2. Find an old Mac machine running macOS 10.11 to build libs, which will force everything to be build against this OS.

I tried "--build-from-source ", but it does not work for me.
